internal/model/plugin_test.go

```diff
@@ -137,6 +137,7 @@ func TestPluginsToWriter(t *testing.T) {
 			ReleaseStage:    Production,
 			Manifest: &mattermostModel.Manifest{
 				Id:      "demo",
+				Name:    "Demo",
```

Comment:
Since https://github.com/mattermost/mattermost-server/pull/16438 a `Name` is required